### ISSUE TITLE
TracingConfig Instantiation in 3.8.0

### ISSUE THREAD

**Issue by vbakaev** [MonsantoCo/cloudformation-template-generator#233]
Can you check `com.monsanto.arch.cloudformation.model.resource.TracingConfig` please,
 it should be a case class?
I have a problem to create an instance of it.
```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at cloudformation.StackTemplateFormation2$.main(StackTemplateFormation2.scala:11)
	at cloudformation.StackTemplateFormation2.main(StackTemplateFormation2.scala)
Caused by: java.lang.RuntimeException: Cannot automatically determine case class field names and order for 'com.monsanto.arch.cloudformation.model.resource.TracingConfig', please use the 'jsonFormat' overload with explicit field name specification
	at spray.json.ProductFormats.extractFieldNames(ProductFormats.scala:85)
	at spray.json.ProductFormats.extractFieldNames$(ProductFormats.scala:67)
	at com.monsanto.arch.cloudformation.model.resource.TracingConfig$.extractFieldNames(Lambda.scala:231)
	at spray.json.ProductFormatsInstances.jsonFormat1(ProductFormatsInstances.scala:23)
	at spray.json.ProductFormatsInstances.jsonFormat1$(ProductFormatsInstances.scala:22)
	at com.monsanto.arch.cloudformation.model.resource.TracingConfig$.jsonFormat1(Lambda.scala:231)
	at com.monsanto.arch.cloudformation.model.resource.TracingConfig$.<init>(Lambda.scala:237)
	at com.monsanto.arch.cloudformation.model.resource.TracingConfig$.<clinit>(Lambda.scala)
	... 2 more
Caused by: java.lang.RuntimeException: Case class com.monsanto.arch.cloudformation.model.resource.TracingConfig declares additional fields
	at scala.sys.package$.error(package.scala:27)
	at spray.json.ProductFormats.extractFieldNames(ProductFormats.scala:79)
	... 9 more
```
Thank you!

**Comment by seanmcl** [comment]
I just got this too.  Looking.

**Comment by bkrodgers** [comment]
Just released v3.8.1 which includes this fix.  Let me know if you still have issues.
